frontend/src/types/styled-components/index.d.ts

```diff
@@ -0,0 +1,8 @@
+import { theme } from "@/theme";
+import "styled-components";
+
+type USHSTheme = typeof theme;
+
+declare module "styled-components" {
+  type DefaultTheme = USHSTheme;
+}
```

Comment:
Fixes some type errors for theme: https://styled-components.com/docs/api#create-a-declarations-file